webapp/assets/_new/styles/resets.scss

```
@@ -1,7 +1,118 @@
-* {
+// Fonts (previously loaded via styleguide system.css)
+@font-face {
+  font-family: LatoWeb;
+  src: url('/fonts/Lato-Regular.1f440a46.woff2') format('woff2'),
+       url('/fonts/Lato-Regular.ffb25090.woff') format('woff');
+  font-style: normal;
+  font-weight: 400;
+  text-rendering: optimizeLegibility;
```

Stylelint (17.4.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)